js/node/lib/version.ts

```diff
@@ -4,4 +4,4 @@
 // This file is generated by /js/scripts/update-version.ts
 // Do not modify file content manually.
 
-export const version = '1.17.3';
+export const version = '1.17.3-rev.1';
```

Comment:
not sure if rev.1 is needed since 1.17.3 never made it to npmjs.org

Reply:
1.17.3 is already published.